Dockerfile

```diff
@@ -54,4 +54,9 @@ VOLUME ["/workspace"]
 #"/var/run/docker.sock" needs to be mounted for container based transformers to work with docker
 # Start app
 WORKDIR /workspace
+
+RUN microdnf install -y openssh openssh-clients
+RUN mkdir -p $HOME/.ssh
+RUN printf "bitbucket.org ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIIazEu89wgQZ4bqs3d63QSMzYVa0MuJ2e2gKTKqu+UUO\ngithub.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==\ngitlab.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf\ngitlab.com ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCsj2bNKTBSpIYDEGk9KxsGh3mySTRgMtXL583qmBpzeQ+jqCMRgBqB98u3z++J1sKlXHWfM9dyhSevkMwSbhoR8XIq/U0tCNyokEi/ueaBMCvbcTHhO7FcwzY92WK4Yt0aGROY5qX2UKSeOvuP4D6TPqKF1onrSzH9bx9XUf2lEdWT/ia1NEKjunUqu1xOB/StKDHMoX4/OKyIzuS0q/T1zOATthvasJFoPrAjkohTyaDUz2LN5JoH839hViyEG82yB+MjcFV5MU3N1l1QL3cVUCh93xSaua1N85qivl+siMkPGbO5xR/En4iEY6K2XPASUEMaieWVNTRCtJ4S8H+9\ngitlab.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFSMqzJeV9rUzU4kWitGjeR4PWSa29SPqJ1fVkhtj3Hw9xjLVXVYrU9QlYWrOLXBpQ6KWjbjTDTdDkoohFzgbEY=\nbitbucket.org ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAubiN81eDcafrgMeLzaFPsw2kNvEcqTKl/VqLat/MaB33pZy0y3rJZtnqwR2qOOvbwKZYKiEO1O6VqNEBxKvJJelCq0dTXWT5pbO2gDXC6h6QDXCaHo6pOHGPUy+YBaGQRGuSusMEASYiWunYN0vCAI8QaXnWMXNMdFP3jHAJH0eDsoiGnLPBlBp4TNm6rYI74nMzgz3B9IikW4WVK+dc8KZJZWYjAuORU3jc1c/NPskD2ASinf8v3xnfXeukU0sJ5N6m5E8VLjObPEO+mN2t/FZTMZLiFqPWc/ALSqnMnnhwrNi2rbfg/rd/IpL8Le3pSBne8+seeFVBoGqzHM9yXw==" > $HOME/.ssh/known_hosts
```

Comment:
Can you please explain this part? Do we really need to hardcode the public keys?

Reply:
Those public keys are hard coded in the codebase, in the spirit of consistency I added them there too but I am more than happy to remove them :)

Reply:
Do we need to install `ssh` ? I think we use a library to do the cloning https://github.com/konveyor/move2kube/blob/aa79dfac86bcfd50e44911f8a65bc5733b08372c/common/vcs/git.go#L29
@kmehant

Reply:
@gabriel-farache Does this change support cloning from Github Enterprise? like `github.mycompany.com` ?

Reply:
https://github.com/konveyor/move2kube/pull/1126#discussion_r1441838203

`go-git` library internally can pick up the SSH agent on the host/container for remote repository pulling.

Reply:
https://github.com/konveyor/move2kube/pull/1126#discussion_r1442562217

AFAIK, for each such private host, a separate record has to go into the known_hosts file as well.

Reply:
https://github.com/konveyor/move2kube/pull/1126#discussion_r1444798459

@gabriel-farache I have not tested it, but, as I mentioned here (https://github.com/konveyor/move2kube/pull/1126#discussion_r1442791080) go-git would pick up ssh agent from the host/container and thanks for adding openssh to the Dockerfile so go-git should now be able to pick it up.

Reply:
@kmehant if it's tested and working we can merge this PR.

Reply:
@HarikrishnanBalagopal just tested it, we need to start the ssh-agent and initing the .ssh/ keys.

For which we need to run
```
eval "$(ssh-agent)" && ssh-add
```

This should go after spawning the container with .ssh folder mounted, so should we be adding that to docs somwhere? if so let me merge this PR.